components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa/src/Publisher/Models/Events/PendingAlarmModel.cs

```diff
@@ -0,0 +1,99 @@
+﻿namespace Microsoft.Azure.IIoT.OpcUa.Publisher.Config.Models.Events {
+    using System;
+    using System.Collections.Generic;
+    using System.Runtime.Serialization;
+
+    /// <summary>
+    /// Settings for pending alarms feature
+    /// </summary>
+    [DataContract]
+    public class PendingAlarmModel {
+        /// <summary>
+        /// Is pending alarms enabled for this event node?
+        /// </summary>
+        [DataMember(EmitDefaultValue = true)]
+        public bool Enabled { get; set; } = false;
+
+        /// <summary>
+        /// Time interval for sending pending interval updates
+        /// </summary>
+        [DataMember(EmitDefaultValue = true)]
+        public int? UpdateInterval { get; set; } = 30000;
+
+        /// <summary>
+        /// UpdateInterval as TimeSpan.
+        /// </summary>
+        [IgnoreDataMember]
+        public TimeSpan? UpdateIntervalTimespan {
+            get => UpdateInterval.HasValue ?
+                TimeSpan.FromMilliseconds(UpdateInterval.Value) : (TimeSpan?)null;
+            set => UpdateInterval = value != null ?
```

Comment:
Are these setters used?

Reply:
Maybe not, don't know yet. Will remove if not needed.

Reply:
Removing them for now, don't see them being used.